docs/userguide/icu_data/buildtool.md

```diff
@@ -104,6 +104,23 @@ The default configuration is equivalent to a filter file like this:
       ]
     }
 
+If you do want to include these collation tailorings,
+you can configure ICU with a filter file which has the opposite effect:
+
+    {
+      "resourceFilters": [
+        {
+          "categories": [
+            "coll_tree"
+          ],
+          "rules": [
+            "+/collations/big5han",
+            "+/collations/gb2312han"
```

Comment:
Does `+/collations/*` work?

I assume you did this because `+/collations` does not work?

Reply:
If the user wants to include "all" collations, they should get them all, even if ICU starts suppressing even more of them in the future

Reply:
The default is to include everything, with this new exception. The filter that I provided here undoes the exception. I think it makes sense as is, including “If you do want to include **these** collation tailorings, ...” (emphasis for comment).

Reply:
I just verified that just `+/collations` works as well. I wouldn't replace what I have, but I could add that as an alternative, with an explanatory paragraph. Which way do you want to go?

Reply:
I just amended the docs to add a paragraph with alternate rules but without spelling out the whole filter file with each of those alternatives.

If we can't settle this before the ICU 73 code freeze, then we need to punt it to a post-73 doc fix.